### PR TITLE
add remarkgfm plugin

### PR DESCRIPTION
**Pull request by InsaneZein**

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -1,2 +1,7 @@
 // Importing Global Variables
 @import "~@redhat-cloud-services/frontend-components-utilities/styles/_all";
+
+h1 {
```
Comment (Hyperkid123) [inline src/App.scss]: Is there any reason for this custom styling?

File: src/App.js [hunk 2]
```diff
@@ -16,7 +19,88 @@ const App = () => {
     };
   }, []);
 
-  return <Routes />;
+  const components = {
+    h1: (props) => {
```
Comment (Hyperkid123) [inline src/App.js:23]: The `h` tags should use the `Title` component from PF: https://www.patternfly.org/v4/components/title

File: src/App.scss (hunk 1 above)
Comment (InsaneZein) [reply src/App.scss]: oh that wasn't supposed to make it here. I was just experimenting with something. I've removed it.

File: src/App.js (hunk 2 above)
Comment (InsaneZein) [reply src/App.js:23]: fixed

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**